docs/changelog/v1.3.0.md

```diff
@@ -0,0 +1,11 @@
+# v1.3.0
+
+## Changes
+
+* plugin: add flag to ignore removals [#1166](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1166)
+* operator: fix replacements with empty monitor conf setting [#1141](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1141)
+* operator: make use of constants for storage engine references [#1165](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1165)
+* operator: Update ginkgo version to v2 [#1156](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1156)
+* plugin: skip Pods that are part of a process group marked for removal [#1132](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1132)
+* operator: Ensure we delete all resources when a process group is in resource terminating state [#1145](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1145)
+* plugin: experimental command for the plugin to show the state of exclusions [#1133](https://github.com/FoundationDB/fdb-kubernetes-operator/pull/1133)
```

Comment:
I think grouping them with a header is more readable (or at least group all operator changes together and all plugin changes).